compiler/rustc_borrowck/src/diagnostics/region_errors.rs

```diff
@@ -777,6 +777,7 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
                 Some(arg),
                 captures,
                 Some((param.param_ty_span, param.param_ty.to_string())),
+                self.infcx.tcx.is_suitable_region(f).map(|r| r.def_id).unwrap(),
```

Comment:
Instead of unwrapping, can we make the argument an `Option`?

Reply:
Sure, changed to `Option`.